src/main/java/io/anserini/eval/Qrels.java

```diff
@@ -204,4 +204,25 @@ public static boolean contains(Path topicPath) {
     }
     return false;
   }
+
+  public static boolean containsSymbol(Path topicPath) {
+    for (Qrels c: Qrels.values()) {
```

Comment:
looping seems inefficient?

Why not do it once (on init), and build a dictionary (e.g., Map) to support quick lookup?